app/.gitignore

```
@@ -29,5 +27,6 @@ l10n/locales/*/messages-compiled.json
 # Sentry Config File
 .env.sentry-build-plugin
 
-# json-autotranslate
-.json-autotranslate-cache
\ No newline at end of file
+# autotranslate
+.json-autotranslate-cache
+*.template.json
```

Make them specific to the directory, please, e.g.:
```suggestion
l10n/locales/*/*.json-autotranslate-cache
l10n/locales/en/messages.template.json
```
or similar

---

`.json-autotranslate-cache` is a generated directory in `app/`